src/sonic-config-engine/tests/sample_hwsku.json

```diff
@@ -1,7 +1,7 @@
 {
     "interfaces": {
         "Ethernet0": {
-            "default_brkout_mode": "1x100G[40G]"
+            "default_brkout_mode": "4x25G[10G]"
```

Comment:
what's the intention here?

Reply:
These ports Ethernet1 and Ethernet2 were defined as part of port channel in simple-sample-config.xml. So to align with that I modified these to be in 4x25G[10G] mode.